lib/pb_extern.h

```diff
@@ -6,6 +6,10 @@
 /// `PUBNUB_EXTERN` before including any Pubnub header.
 
 #ifndef PUBNUB_EXTERN
+#ifdef __cplusplus
+#define PUBNUB_EXTERN extern "C"
```

Comment:
As the CI tests are using the Makefile version of the build system it is likely this define change which is breaking the CI

Reply:
Not sure how it would break. Surely exposing C interfaces should be demangled?

Reply:
But anyway it is a little but strange as our unit tests are written in C and compiled by gcc so the behaviour shouldn't change. ;o